packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -9,6 +9,13 @@ import type { Props } from "./types";
 import Header from "../components/Header";
 import Expandable from "./components/Expandable";
 import handleKeyDown from "../../utils/handleKeyDown";
+import Stack from "../../Stack";
+
+const Actions = ({ actions }) => (
```

Comment:
I created this small component because the of avoiding the code duplication as it's used twice within this file.